src/riak_kv_ts_util.erl

```diff
@@ -669,17 +669,35 @@ check_table_feature_supported(DDLRecCap, DecodedReq) ->
                     {error, create_table_not_supported_message(Table)}
             end;
         {ok, _, {_, Table}} ->
-            case is_table_supported(DDLRecCap, Table) of
+            case is_table_active_and_supported(DDLRecCap, Table) of
```

Comment:
Happy to rename it and restructure it.  It's all how much we want to refactor in this PR.